boards/msp_exp432p401r/Makefile

```diff
@@ -19,7 +19,7 @@ flash-debug: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).elf
 	$(OPENOCD) $(OPENOCD_OPTIONS) -c "init; reset halt; flash write_image erase $<; verify_image $<; reset; shutdown"
 
 .PHONY: flash
-flash: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
+flash: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).bin
```

Comment:
Is this change needed?

Reply:
During testing I noticed that flashing the kernel using `make flash` overwrites the .apps section, which requires to also flash the application again. This is probably the case, because the elf file contains a 0-byte as a placeholder in section .apps. As far as I understand the .bin file doesn't contain this placeholder. Therefore I changed the Makefile to flash the binary instead of the elf file, which allows to flash the kernel independently from the application.